clang/test/Driver/cl-zc.cpp

```diff
@@ -123,6 +123,15 @@
 // CHECK-CHAR8_T_: "-fno-char8_t"
 
 
+// RUN: %clang_cl /TC /dev/null /E -Xclang -dM 2> /dev/null | FileCheck -match-full-lines %s --check-prefix=NOSTDC
```

Comment:
Driver tests usually only do `-###` and check the generated cc1 options. This command runs the preprocessor, which can be moved to clang/test/Preprocessor using `%clang_cc1`

Reply:
Moved to clang/test/Preprocessor, thanks!